### DISCUSSION TITLE
Training Interruption at Epoch 54

### DISCUSSION THREAD

**Discussion by NikhilKalloli** [THU-MIG/yolov10#325] [Q&A] [answered]
Hello,

My model training consistently gets interrupted at epoch 54 on both Google Colab and my local system. Has anyone else faced this issue or have any suggestions for resolving it?

Thanks!

**Comment by sorohere** [comment, marked as answer]
It sounds like your training process might be exceeding your local resources. you could try using smaller pre-trained models of the same type to train on your custom dataset. 
Here's a link to find all such models: _[https://docs.ultralytics.com/models/yolov10/#model-variants](url)_
